### PR TITLE
openapi: Add display attributes for SSH plugin

### PR DESCRIPTION
**Pull request by averche**
Please see https://github.com/hashicorp/vault/pull/19319 for more details on how this will affect the generated OpenAPI schema.

____

### The following OperationID's will be generated for SSH plugin (17 total)

| Path | Operation ID |
| ---- | ------------ |
| "{ssh_mount_path}/config/ca" | ssh-read-ca-configuration |
| | ssh-configure-ca |
| | ssh-delete-ca-configuration |
| "{ssh_mount_path}/config/zeroaddress" | ssh-read-zero-address-configuration |
| | ssh-configure-zero-address |
| | ssh-delete-zero-address-configuration |
| "{ssh_mount_path}/creds/{role}" | ssh-generate-credentials |
| "{ssh_mount_path}/issue/{role}" | ssh-issue-certificate |
| "{ssh_mount_path}/lookup" | ssh-list-roles-by-ip |
| "{ssh_mount_path}/public_key" | ssh-read-public-key |
| "{ssh_mount_path}/roles" | ssh-list-roles |
| "{ssh_mount_path}/roles/{role}" | ssh-read-role |
| | ssh-write-role |
| | ssh-delete-role |
| "{ssh_mount_path}/sign/{role}" | ssh-sign-certificate |
| "{ssh_mount_path}/tidy/dynamic-keys" | ssh-tidy-dynamic-host-keys |
| "{ssh_mount_path}/verify" | ssh-verify-otp |

### CODE REVIEW

**Comment by cipherboy** [comment]
Could `ssh-sign` and `ssh-issue` include certificate in the name by any chance? I think that'll make them clearer.

**Comment by averche** [comment]
> Could `ssh-sign` and `ssh-issue` include certificate in the name by any chance? I think that'll make them clearer.

Will update, thanks!

**Review by dhuckins — APPROVED** [review]

File: builtin/logical/ssh/path_cleanup_dynamic_host_keys.go [hunk 1]
```diff
@@ -13,6 +13,11 @@ const keysStoragePrefix = "keys/"
 func pathCleanupKeys(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "tidy/dynamic-keys",
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixSSH,
+			OperationVerb:   "tidy",
+			OperationSuffix: "host-keys",
```
[the remark this answers is not in the document]
Comment (averche) [reply builtin/logical/ssh/path_cleanup_dynamic_host_keys.go:19]: I was following the [documentation](https://developer.hashicorp.com/vault/api-docs/secret/ssh#tidy-host-keys) on this one. Perhaps `"dynamic-host-keys"` for extra clarity?

[the remark this answers is not in the document]
Comment (cipherboy) [reply builtin/logical/ssh/path_cleanup_dynamic_host_keys.go:19]: This is mostly as a clean up for deprecated functionality and so not oft used, I think `ssh-tidy-dynamic-host-keys` is perhaps best here.

File: builtin/logical/ssh/path_verify.go [hunk 2]
```diff
@@ -11,6 +11,10 @@ import (
 func pathVerify(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "verify",
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixSSH,
+			OperationVerb:   "verify",
```
Comment (cipherboy) [inline builtin/logical/ssh/path_verify.go:16]: One nit: could we make this `verify-otp`? it doesn't work for SSH CA certs.

**Review by cipherboy — APPROVED** [review]
Looking good!

**Pull request merged**